frontend/src/components/config/Theme.tsx

```diff
@@ -13,8 +15,14 @@ export const ThemeConfig: any = {
     gray: 'gray',
     lightGray: 'lightgray',
     white: 'white',
+    greenCircle: 'linear-gradient(207.68deg, #14D633 10.68%, #DAFFB5 91.96%)',
+    redCircle: 'linear-gradient(207.68deg, #DD145D 10.68%, #FFB7B7 91.96%)',
+    blueCircle: 'linear-gradient(207.68deg, #133ED7 10.68%, #B7D4FF 91.96%)',
+    pinkCircle: 'linear-gradient(207.68deg, #F25AFF 10.68%, #F5DAFF 91.96%)',
+    yellowCircle: 'linear-gradient(207.68deg, #FFC700 10.68%, #FFFDC1 91.96%)',
+    purpleCircle: 'linear-gradient(207.68deg, #9845EC 21.37%, #D9B4FF 102.65%)',
```

Comment:
Maybe rename these to just `color` without the circle part since we're using them not just for the circles? Or maybe we could move them into like a nested gradients object or something (i.e. `theme.colors.gradients.blue`)